terraform/modules/datapump/cloudwatch.tf

```diff
@@ -30,7 +30,7 @@ resource "aws_cloudwatch_event_target" "sync-areas" {
 
 resource "aws_cloudwatch_event_target" "sync-deforestation-alerts" {
   rule      = aws_cloudwatch_event_rule.everyday-7-pm-est.name
-  target_id = substr("${local.project}-sync-deforestation-alerts${local.name_suffix}", 0, 64)
+  target_id = substr("${local.project}-nightly-sync-areas${local.name_suffix}", 0, 64)
```

Comment:
I think this may be a merge issue, I changed the names in a different branch to be clear. Now it looks like you use the same name over and over.

Reply:
D'oh. Yep, looks like I mixed-up a merge conflict.